apps/testingaccessibility/src/pages/thanks/purchase.tsx

```diff
@@ -76,11 +85,23 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
     purchaseType = NEW_INDIVIDUAL_PURCHASE
   }
 
+  let bulkCouponRedemptionPurchase: object | null = null
+  if (purchase.bulkCoupon?.id) {
+    const {getUserByEmail, getBulkCouponRedemptionPurchase} = getSdk()
+    const user = await getUserByEmail(email)
+    bulkCouponRedemptionPurchase = await getBulkCouponRedemptionPurchase(
+      purchase.bulkCoupon.id,
+      user?.id || null,
+    )
+  }
```

Comment:
Use a couple database functions from `getSdk` to look up the user ID (they may not be logged in at this point) and then determine if this person has self-redeemed one of the seats.